monkey/monkey_island/cc/ui/src/components/report-components/RansomwareReport.js

```diff
@@ -4,9 +4,15 @@ import ReportHeader, {ReportTypes} from './common/ReportHeader';
 import ReportLoader from './common/ReportLoader';
 import pluralize from 'pluralize'
 import FileEncryptionTable from './ransomware/FileEncryptionTable';
+import NumberedReportSection from './ransomware/NumberedReportSection.tsx';
 
 import '../../styles/pages/report/RansomwareReport.scss';
 
+const LATERAL_MOVEMENT_DESCRIPTION = 'After the initial breach, the attacker will begin the Lateral \
+                                      Movement phase of the attack. They will employ various \
+                                      techniques in order to compromise other systems in your \
+                                      network in an attempt to seek out your digital crown jewels.'
```

Comment:
The ransomware is all about encryption and ransom, not necessarily about "seeking out crown jewels". I would rephrase to "<...> They will use various techniques to spread in the network and encrypt as many files as possible".